src/components/30-organisms/footer/index.js

```diff
@@ -20,6 +20,37 @@ const _setMaxHeightToZero = panel => {
   panel.style.maxHeight = '0px';
 };
 
+// hence there is no re-render to set the caret-open-class
```

Comment:
This comment can not be understood properly. Rephrase: Caret state change *bypasses* render() and directly manipulates dom-element classes on purpose here, because the height measurement of open accordion items conflicted with a model-based always-rerender approach in the past. TODO: Explore how to get rid of height measurement or make it compatible with always-rerender, then refactor and remove this code in the future.